setup.cfg

```diff
@@ -71,7 +71,7 @@ install_requires =
   enrich>=1.2.6
   packaging
   pyyaml
-  rich>=9.5.1
+  rich>=10.7.0
```

Comment:
Version 10.7.0 is barely four months old, so I would rather see that we keep the old constraints and add a compatibility layer to the code that uses console groups.